### DISCUSSION TITLE
How to set a default button style

### DISCUSSION THREAD

**Discussion by groucho86** [zauberzeug/nicegui#1077] [Q&A] [answered]
### Question

I'm able to override default font with
`ui.query('body').style('font-family: aFont') `

Is there a similar way to globally disable/change buttons' style? I know this can be done  on a per button-basis `ui.button.style(text-transform: "none")` but can it be done globally?

**Comment by falkoschindler** [comment, marked as answer]
Hi @groucho86!
To globally define CSS style, you can use `ui.add_head_html`:

```py
ui.add_head_html('''
<style>
    .q-btn {
        text-transform: none;
    }
</style>
''')

ui.button('Click me', on_click=lambda: ui.notify('Hello world!'))
```

Note that this selects elements with the class "q-btn" instead of all button tags. The reason for this choice is a bit technical (to increase the specificity of the CSS selector), but basically you can try out what works and what not. As a last resort you can enforce a certain style with the "!important" suffix.
